client/blocks/jetpack-backup-creds-banner/index.jsx

```diff
@@ -0,0 +1,60 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Banner from 'components/banner';
+import getRewindState from 'state/selectors/get-rewind-state';
+import QueryRewindState from 'components/data/query-rewind-state';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSiteSlug, isJetpackSite } from 'state/sites/selectors';
+
+class JetpackBackupCredsBanner extends Component {
+	static propTypes = {
+		// Banner location, passed to tracks event, single word to follow tracks naming conventions.
+		location: PropTypes.string.isRequired,
```

Comment:
Nit: it's also common to list the connected props, too, under their own `// Connected props` section.